rotkehlchen/tests/api/test_history_events_export.py

```diff
@@ -173,7 +173,7 @@ def test_history_export_csv_custom_delimiter(
         api_url_for(rotkehlchen_api_server_with_exchanges, 'exporthistoryeventresource'),
         json={'async_query': False, 'directory_path': str(csv_dir)},
     )
-    assert_csv_export_response(response, csv_dir, csv_delimiter=csv_delimiter)
+    assert_csv_export_response(response, csv_dir / 'historyevents_till_20250430.csv', csv_delimiter=csv_delimiter)  # noqa: E501
```

Comment:
I want to see also a type/subtype query filename test

Reply:
added one.